src/game.h

```diff
@@ -419,16 +419,19 @@ class Game
 	void updateCreatureWalkthrough(const Creature* creature);
 	void updateKnownCreature(const Creature* creature);
 
-	GameState_t getGameState() const;
-	void setGameState(GameState_t newState);
-	void saveGameState();
+	bool inState(GameState_t state) const
+	{
+		return gameState == state;
+	}
```

Comment:
Same, you can drop it and move the definition of `getState` to the header so it gets inlined (likely gets inlined anyway with LTO)